### ISSUE TITLE
Seems like the latest Chrome for Windows Version 86.0.4240.198 (Official Build) (64-bit) broke it

### ISSUE THREAD

**Issue by davidlkent** [grantwinney/hide-google-doodles#25]
Tried various hacks such as completely clearing cache and data, but cannot make the proper behavior return. You are a hero for creating this extension so I hope it's not too much work to fix it. Thanks, - David

**Comment by grantwinney** [comment]
Hey David, thanks for letting me know. Out of curiosity, is this in Chrome on the "new tab" page? Or when you actually visit google.com? Because it never worked on the former, but still does (I think!) on the latter.

But google seems to have made changes that cause it to be showed in the "new tab" even where it wouldn't have before...

**Comment by davidlkent** [comment]
Hey Grant, thanks again for this one-of-a-kind necessity.

 

I always use google.com because I hate their New Tab page, too. I use the extension New Tab Redirect by Jim Schubert to accomplish that automatically. Note that in the latest Firefox 82.0.3 (64-bit) that same combination is still working, which is interesting.

 

I also enable “Show home button” and set the link to be https://www.google.com/.  Opening a tab and pasting that URL directly into the address bar has no effect – the dreaded doodle is back.

 

Good luck with this one, - David

**Comment by davidlkent** [comment]
Here’s a clue: when I sign out of Google it starts working again. Sign back in under any account and the doodle comes back. - David

**Comment by grantwinney** [comment]
I setup Chrome with just this extension and that new tab redirect one, and I've tried logging into and out of my google account, and I can't for the life of me recreate this. It's frustrating.

I suppose if you've got time sometime, when you notice the doodle, it might help if you right-click and select 'view page source' (or just press ctrl+u), and then paste the html code somewhere. Right here is fine I suppose, or [pastebin](https://pastebin.com/) or something. This extension expects certain elements to be there, so it can replace them.. maybe sometimes they're not loading or their named differently than I expect? Just not sure yet...

**Comment by davidlkent** [comment]
I appreciate your perseverance. Yes, I’ll be happy to help diagnose. I sure hope I’m not the only one with this problem. - David

**Comment by davidlkent** [comment]
Here’s a screen shot

 

http://my.jetscreenshot.com/20266/20201114-fle2-87kb

 

View Source and Outer HTML are attached

 

Good luck, - David

**Comment by grantwinney** [comment]
No problem David. I see the screenshot but not the other files. If you just replied to the email GitHub sends, and attached the files to your email, I'm not sure GitHub actually does anything with them... it'd be nice if they attached them to the issue, but they don't seem to be.

You could probably just paste them into GitHub's gist site here -> https://gist.github.com

You might also be able to drag them right into the editor pane on GitHub's site if you rename them with a ".txt" extension... maybe.

**Comment by davidlkent** [comment]
Ok, Grant – I created this public gist

 

https://gist.github.com/davidlkent/59a1243e6af65fdca8f1a77f5145133e#file-view-source_https___www-google-com-html

 

I tried to make both wrap but the second file is one long line, despite best effort.  Hopefully you can format as HTML for analysis.

 

Good luck, and thanks. - David

**Comment by grantwinney** [comment]
Thanks for the help, David. That source code helped.. it showed that the element I was trying to replace on the page wasn't necessarily always present. I published some changes today (v 1.3.4) that should hopefully fix that.

**Note:** Because I added a new google URL to the manifest.json for it to check, Chrome seems to be disabling the addon after the update... if you get the update and re-enable the extension, it should work better now. If you get a chance to try it, let me know how it goes.
